src/components/molecules/InteractiveModalImage.module.css

```diff
@@ -31,10 +36,10 @@
 #close:after {
   position: absolute;
   top: 15%;
-  left: calc(50% - 0.0625em);
-  width: 0.125em;
+  left: calc(50% - 0.0625rem);
+  width: 0.125rem;
   height: 70%;
-  border-radius: 0.125em;
+  border-radius: 0.125rem;
```

Comment:
suggestion(blocking): I would prefer having this number in a css variable. Can be scoped to only this close btn if we don't use it anywhere else, e.g.
```css
#close:after {
  --close-width: 0.125rem;
  width: var(--close-width);
  left: calc(var(--close-width) / 2);
}
```